libevmasm/AssemblyItem.h

```diff
@@ -250,6 +292,13 @@ class AssemblyItem
 	AssemblyItemType m_type;
 	Instruction m_instruction; ///< Only valid if m_type == Operation
 	std::shared_ptr<u256> m_data; ///< Only valid if m_type != Operation
+
+	struct FunctionSignature
+	{
+		uint8_t argsNum;
+		uint8_t retsNum;
+	};
+	std::optional<FunctionSignature> m_functionSignature; ///< Only valid if m_type == CallF or JumpF
```

Comment:
This should have a getter, similarly to `instruction()`, with an assert that the item type allows it.

Reply:
Done